src/cmd/cmdsim.rb.in

```diff
@@ -178,6 +178,9 @@ COMMANDS = { 'sim' =>
   "  --version                    Print Gazebo version information.                \n"\
   "\n"\
   "  -z [arg]                     Update rate in Hertz.                            \n"\
+  "\n"\
+  "  --block-on-sdf-errors [arg]  Stop loading the server in presence of SDF       \n"\
```

Comment:
I don't know if we should make this available for end users. When I was suggesting the feature, I only had our unit tests in mind where we would use it only in a narrowly scoped use case (invalid mesh path). My concern is that users can now use this flag and would be confused when things don't work as expected. Is there an actual use case for enabling this?

Reply:
More of a nice to have. I have sdf files with a few uri paths that don't exist on my filesystem and this lets me launch gazebo with only the models with valid URIs instead of exiting. I can also just comment them out.

I removed the changes here. We can revisit if we have a better or more common use case.